javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/AgentInstaller.java

```diff
@@ -349,6 +352,18 @@ public <T> void customize(
         });
   }
 
+  private static void addSqlCommenterCustomizers(ClassLoader extensionClassLoader) {
```

Comment:
Based on `addHttpServerResponseCustomizers` that is right before this method. It supports multiple customizers but really probably works best when there is only once customizers, since since the customizers would most likely end up overwriting what the previous customizer has set.